### PR TITLE
[4.0] Debug User/Group Views fix

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue #13910 .

### Summary of Changes
Fixes icon colors. Allows table column titles to wrap. Adds `table-responsive` class allowing horizontal scrolling.

### Testing Instructions
Apply patch and open... 
`administrator/index.php?option=com_users&view=debuguser&user_id=<your_user_id>` (for your user)
`administrator/index.php?option=com_users&view=debuggroup&group_id=8` (for super user group)

### Before patch
debug1

### After patch
debug2

### Documentation Changes Required
None

### CODE REVIEW

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on ca67d152161c24c1662426c242c05b7e4d4e6dc0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14255">issues.joomla.org/tracker/joomla-cms/14255</a>.</sub>

File: administrator/templates/atum/scss/pages/_com_users.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// com_users
+
+.com_users {
+  &.view-debuggroup, &.view-debuguser {
+    thead th {
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/pages/_com_users.scss]: are you able to remove the `thead`?

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on ca67d152161c24c1662426c242c05b7e4d4e6dc0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14255">issues.joomla.org/tracker/joomla-cms/14255</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14255">issues.joomla.org/tracker/joomla-cms/14255</a>.</sub>

**Pull request merged**